File: lessons/compiled_lang/go/src/j20.arguments.go
package main

import (
	"fmt"
	"os"
)

func main() {
	args := os.Args[1:]

	fmt.Println("The arguments passed are (reverse order):")
	for i := len(args) - 1; i >= 0; i-- {
		fmt.Printf(" item %d: %s\n", i+1, args[i])
	}
}

File: lessons/compiled_lang/cpp/src/j20.arguments.cpp
#include <iostream>

int main(int argc, char* argv[]) {
    std::cout << "The arguments passed are (reverse order):" << std::endl;
    for (int i = argc - 1; i >= 1; i--) {
        std::cout << " item " << i << ": " << argv[i] << std::endl;
    }

    return 0;
}
